docs/mesos/vagrant-centos/Vagrantfile

```diff
@@ -110,6 +110,9 @@ Vagrant.configure("2") do |config|
       # Selinux => permissive
       host.vm.provision :shell, inline: "setenforce permissive"
 
+      # Install docker here, as otherwise it will run 'yum update' and unexpectedly ugrade mesos
+      host.vm.provision :docker
```

Comment:
Calico-node and libnetwork are installed on all machines (master and agents). marathon-lb is the only one that is not on all machines.